CONTRIBUTING.md

```diff
@@ -144,6 +144,39 @@ NOTE: Once you submitted pull request to Cactus repository, step 6 is not necess
 
 NOTE: You can refer original tutorial ['GitHub Standard Fork & Pull Request Workflow'](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
 
+### Directory structure
+
+Whenever you begin to contribute your codes on Hyperledger Cactus, you should follow the directory strecture on Hyperledger Cactus.
+The current directory structure is described as the following:
+
+> - contrib/ : Contributions from each participants, which are not directly dependent on Cactus code.
+>   - Fujitsu-ConnectionChain/
+>   - Accenture-BIF/
+> - docs/
+>   - API/
+>     - business-logic-plugin.md
+>     - ledger-plugin.md
+>     - routing-interface.md
+> - examples/
+>   - example01-car-trade/
+>     - src/
+> - plugins/
+>     - business-logic-plugin/
+>       - lib/ : libraries for building Business Logic Plugin
+>     - ledger-plugin/ : Codes of Ledger Plugin
```

Comment:
@takeutak Is this meant to be a replacement to the existing package structure we have?
https://github.com/hyperledger/cactus/tree/master/packages/cactus-plugin-ledger-connector-quorum
https://github.com/hyperledger/cactus/tree/master/packages/cactus-plugin-ledger-connector-besu